Numbers concept (#1054)
exercism/haskell: exercises/concept/temperature/.docs/hints.md

> # Temperature Exercise

Reviewer: The headings are stripped when this content is used, so we usually just have it be the type of file

```suggestion
# Hints
```

Author: Do they (headings) serve any purpose in the hints file then?

## Convert Farenheit to Celsius

- Temperature in degrees Celsius is defined as: T(°F) = T(°C) × 1.8 + 32

> - The temperature in Farenheit is provided as an `Integer` and must return Celsius as a float. The function `fromInteger` will come in handy.

Reviewer: ```suggestion
- The temperature in Farenheit is provided as an `Integer` and must return Celsius as a float. The `fromInteger` function will come in handy.
```

## Convert Celsius to Farenheit

- Temperature in degrees Fahrenheit is defined as: T(°C) = (T(°F) - 32) / 1.8

> - The temperature in Celcius is provided as `Float` and must return Farenheit as the next closest `Integer`. The function `ceiling` will come in handy.

Reviewer: ```suggestion
- The temperature in Celcius is provided as `Float` and must return Farenheit as the next closest `Integer`. The `ceiling` function will come in handy.
```

# General

> Temperature in degrees Celsius is defined as: T(°F) = T(°C) × 1.8 + 32
> 
> Temperature in degrees Fahrenheit is defined as: T(°C) = (T(°F) - 32) / 1.8

Reviewer: ```suggestion
- Temperature in degrees Celsius is defined as: T(°F) = T(°C) × 1.8 + 32
- Temperature in degrees Fahrenheit is defined as: T(°C) = (T(°F) - 32) / 1.8
```

Reviewer: These two hints should be task-specific hints, as they can be linked to a particular task.

> ## Type conversion

Reviewer: This heading should be the name of the corresponding task. In general, there are two types of tasks: 

- General hints that apply to all tasks
- Task-specific hints

Where possible, task-specific hints are to be preferred. 

See https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-hints-md for documentation on the `hints.md` file.

> Remember the temperature is provided as Integer, but your function will pass it through operations that expect Floats. The function `fromInteger` will come in handy.

Reviewer: Where possible, code should be in backticks:

```suggestion
Remember the temperature is provided as an `Integer`, but your function will pass it through operations that expect `Float`s. The function `fromInteger` will come in handy.
```

### Return Type

> Remember you must return a Float representing the temperature, not a string.

Reviewer: ```suggestion
- Remember you must return a `Float` representing the temperature, not a string.
```
